Assets/HoloToolkit/SpectatorView/Scripts/Utils/TweenAlpha.cs

```diff
@@ -69,6 +69,14 @@ private void Start()
             }
         }
 
+        /// <summary>
+        /// Destroys material before death
```

Comment:
Don't need summary comments on private methods, but I chuckled a bit on this one.